assets/css/main.css

```diff
@@ -19,6 +19,16 @@ h1 {
   font-size: 48px;
 }
 
+.jumbotron{
```

Comment:
Please leave a space between CSS property and  opening brace `{`.

Let's keep the code consistent across codebase.